Authentication/SignUp/ViewModel/SignupViewModel.swift

```diff
@@ -22,9 +22,13 @@ public protocol SignupViewModelType {
     
     var password: MutableProperty<String> { get }
     var passwordValidationErrors: AnyProperty<[String]> { get }
+    var showPassword: MutableProperty<Bool> { get }
```

Comment:
Yes. I liked the new name, because `showPassword` may lead to confusion like true because we should make it show or because it is already shown", etc.